docs/meshstack.openshift.metering.md

```diff
@@ -13,4 +13,35 @@ meshStack supports metering and billing for most common OpenShift versions.
 
 ## Supported Resources
 
+Resources with the following traits are addressable in the [Product Catalog](meshstack.billing-configuration.md#defining-a-custom-product-catalog). Operators can use the traits of these resources to define fine-granular product and pricing rules.
+
 Currently OpenShift metering is based on Pods. They are metered via the configured limit of CPU and memory of their containers.
+
+
+### Pod
+<!--snippet:mesh.kraken.productcatalog.traits.kubernetes.pod#type-->
+
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Dhall Type-->
+```dhall
+let PodResourceTraits =
+    {-
+      milliCpu:
+        Configured total CPU limit of all containers in this Pod, in milli CPU (1/1000 of a CPU).
+
+      ramMb:
+        Configured total RAM limit of all containers in this Pod, in MiB.
+
+      status:
```

Comment:
d: We don't have a status predicate defined in our product catalog for pods. But as the status is available in teh traits, i guess it should work

Reply:
While reviewing the trait definition I thought it was an oversight that it was not documented

Reply:
yeah, i am not sure which statuses are available for pods, but it makes sense to provide the option of filtering on status